docs/src/main/asciidoc/kubernetes-client.adoc

```diff
@@ -220,3 +220,47 @@ public class ResourceWatcher<T extends HasMetadata> implements Watcher<T> {
 client.pods().watch(new ResourceWatcher<Pod>());
 ----
 
+== Access to the Kubernetes API
+
+You will most likely need access to the kubernetes API using a ServiceAccount, Role and RoleBinding to run the new application.
```

Comment:
Perhaps we should say: `In many cases in order to access the Kubernetes API server a ServiceAccount, Role and RoleBinding will be necessary`